src/Typeahead.svelte

```diff
@@ -228,18 +236,23 @@
     background-color: #cacaca;
   }
 
+  .active,
+  li:not(:last-of-type).active {
+    border: #0f62fe 1px solid;
```

Comment:
I'm not keen on using a border for the active state. It causes the text of an active item to shift. I'd prefer keeping the existing style of using a background color to distinguish the active but not selected state.

Reply:
Great, what about this blue: #d8e9f3
![image](https://user-images.githubusercontent.com/3764089/108635737-48728500-7481-11eb-9ba9-4d19668ead55.png)